ios/FluentUI.Demo/FluentUI.Demo/Demos/CommandBarDemoController.swift

```diff
@@ -334,6 +339,14 @@ class CommandBarDemoController: DemoController {
         }
     }
 
+    func createHorizontalStackView() -> UIStackView {
+        let switchView = UIStackView()
+        switchView.axis = .horizontal
+        switchView.alignment = .fill
+        switchView.distribution = .fillProportionally
```

Comment:
make sure it looks fine on larger text mode.

Reply:
Updated to improve large text mode support.